spec/models/participant_profile/ect_spec.rb

```diff
@@ -46,6 +46,36 @@
     it { expect(instance.role).to eq("Early career teacher") }
   end
 
+  describe "#with_completion_date_status_or_declaration?" do
```

Comment:
Well, I just added test cases for all the scenarios when this returns true. 
Therefore, it is assumed that in any other case returns false, otherwise what it would be missing is that new scenario spec that makes it return true.

In any case, I don't mind to add it.